scripts/build.py

```diff
@@ -265,33 +310,19 @@ def _specific_prepare(self):
     def _get_qmake_spec(self):
         return 'win32-msvc'
 
-    def _get_make_cmd(self):
-        return "C:/ProgramData/chocolatey/lib/jom/tools/jom.exe"
-
-    def __init__(self, word_size):
-        Builder.__init__(self, word_size)
-
+    def _get_generator(self):
+        return "Ninja"
 
-class MinGWBuilder(Builder):
-
-    def _specific_prepare(self):
-        # See comment in MSVCBuilder.
-        os.environ['QT_FORCE_STDERR_LOGGING'] = '1'
-
-    def _get_qmake_spec(self):
-        return 'win32-g++'
-
-    def _get_make_cmd(self):
-        return 'mingw32-make'
-
-    def _should_run_checks(self):
-        return False
+    def _strip_debug(self):
+        pass
 
-    def _should_publish(self):
-        return True
+    def bundle_csl2xsb(self):
+        if self.word_size != '64':
+            return
```

Comment:
It makes sense to bundle CSL2XSB for both 32bit and 64bit Windows. Both can be used with X-Plane (with 64bit xswiftbus, of course)